### ISSUE TITLE
ignore specified versions

### ISSUE THREAD

**Issue by dylanegan** [wycats/bundler#69]
Hi,

I lock in my versions in the Gemfile and sometimes I just want to pull in the latest and test against them. Would it make sense to have something like;

gem bundle --ignore-version(s)

which would pull in whatever the latest versions are for each gems so you can test against them and then decide whether or not to keep them. Obviously I can edit my Gemfile and take out the specified versions and git checkout if I don't want the change, but that gets annoying after changing a few. Im wondering if this might at all be useful for other people.

Cheers,

Dylan.

**Comment by carllerche** [comment]
An interesting concept. I would like to postpone this until 0.8.0 unless somebody has a patch?

**Comment by carllerche** [comment]
Such a feature should probably be able to take a list of gems for which to limit the ignore versions flag.

**Comment by dylanegan** [comment]
Yeah, didn't work on anything as it was just an idea this morning at work. Specifying for a list gems sounds good too. Hopefully have some time to look at it later this week.

**Comment by dylanegan** [comment]
Spent some time looking through the source and understanding how the versioning works. It looks like the version for each dependency is set before (as the Gemfile is parsed at that time) the options are passed in to the environment. Does that mean it might be best to ignore the version (if the option is set) as it goes to install the gem? Therefore just override the @version in the dependency.

**Comment by fbernier** [comment]
2 years and a couple month later, I had this oment today where I thought this feature would be very useful for testing a new stack with lots of gems.
